bigbang/archive.py

```diff
@@ -248,6 +250,34 @@ def compute_activity(self, clean=True):
 
         return activity
 
+    def get_froms(self):
```

Comment:
This method can easily be generalised for any address header field. Thus instead of hard wiring it to be "From", lets parse an argument which has "From" as its default value, but can also be changed to, e.g., "Reply-To".

Would it be of value to be able to fully dissect address fields into name, localpart, and domain?